### PR TITLE
Kevin/read me

### PR DESCRIPTION
**Pull request by KevinT001**
Updated README.md file. Let me know what you think and what changes you would like to add or take out!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,36 @@
...
+
+
+##### Credits: Eli Sachs & Kevin Ta
+###### Turing School of Software & Design
+
+
+### <font color="gree">Introduction:</font>
+Welcome players! This game of Battleship was created using Ruby code and is run directly in your terminal command line.
+
+Run `battleship_runner.rb` located in root of the `battleship` folder to begin the game.
```
Comment (easachs) [inline README.md:12]: Can we change this to 'run `**ruby** battleship_runner.rb`?

File: README.md [hunk 2]
```diff
@@ -1 +1,36 @@
...
+  * Fire shot (Ex: D3)
+  * Grid will display one of the following:
+    * Missed shot: "M"
+    * Successful shot: "H"
+    * Sunk ship: "X"
+
+  ##### Screenshots
+  ![Blankboard](/Users/kevinta/Desktop/blankboard.jpg "blank8x8")
+
+  ![Image2](/Users/kevinta/Desktop/battleship.jpg "battleship")
```
Comment (easachs) [inline README.md:36]: These images aren't working for me, I believe one way you can create URL image links by drag/dropping them into one of these comment fields.

**Comment by easachs** [comment]
Looks great to me, just practicing leaving comments on Github. Only thing I would change is the photos in the README which I believe will only work for you currently as they are not hosted anywhere online (see comment in code for details)

File: README.md (hunk 1 above)
Comment (KevinT001) [reply README.md:12]: sure! I'll switch it now

**Pull request merged**